targets/xorg

```diff
@@ -108,6 +108,8 @@ install xorg arch=,xserver-xorg-video-fbdev$ltspackages
 if [ "${ARCH#arm}" = "$ARCH" ]; then
     install arch=,xserver-xorg-video-intel$ltspackages
 fi
+# JMT: eventually wrap this removal with an appropriate test
+remove xf86-video-odroid-c1
```

Comment:
It's actually simpler than it looks (there aren't that many different graphics drivers needed on Chromebooks). You can just mirror what is done on Ubuntu/Debian: fbdev everywhere, intel on non-ARM.

`targets/xiwi` will also need fixing.